src/Features/CSharp/Portable/InitializeParameter/CSharpInitializeMemberFromParameterCodeRefactoringProvider.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.CodeAnalysis.CSharp.InitializeParameter
 {
     [ExportCodeRefactoringProvider(LanguageNames.CSharp, Name = nameof(CSharpInitializeMemberFromParameterCodeRefactoringProvider)), Shared]
     [ExtensionOrder(Before = nameof(CSharpAddParameterCheckCodeRefactoringProvider))]
+    [ExtensionOrder(Before = PredefinedCodeRefactoringProviderNames.Wrapping)]
```

Comment:
i would honestly also just make the code action for these wrapping refactoring have .Priority=Low.  I'm struggling to think of any refactoring i'd want it to have priority over.

Reply:
I tried to parameterize the priority for [CodeActionWithNestedActions](http://source.roslyn.io/#Microsoft.CodeAnalysis.Workspaces/CodeActions/CodeAction.cs,431676bb81cbee71,references) and passed `Priority.Low` for the wrapping code action [here](http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Wrapping/AbstractCodeActionComputer.cs,276), but it did not work. Wondering if we only respect the priority for non-nested code actions?

Reply:
> Wondering if we only respect the priority for non-nested code actions?

That would not surprise me at all :)